draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -375,14 +372,7 @@ using a single or a set of APIs independent of the channels that
 are used to convey the signals. The API framework is required even
 if one single channel is used so that any application on a client can
 consume the signals.
-: There might be many channels to signal the metadata such as
-(non-exhaustive list):
 
-  * TCP options {{?RFC9293}}
-  * UDP Options {{?I-D.ietf-tsvwg-udp-options}}
-  * IPv6 Hop-by-Hop Options ({{Section 4.3 of ?RFC8200}})
-  * QUIC CID mapping
-  * ICMP messages
```

Comment:
these lines were all moved to Section 5.3, "Metadata Negotiation Support", approximately line 514, as the server-to-network per-packet metadata is not related to API, but rather something that is learned (negotiated) by the client talking to its network.